src/coreComponents/dataRepository/Group.hpp

```diff
@@ -1492,6 +1492,7 @@ template< typename T >
 Wrapper< T > & Group::registerWrapper( string const & name,
                                        std::unique_ptr< T > newObject )
 {
+  static_assert( not std::is_base_of< WrapperBase, T >::value, "This function should not be used for `WrapperBase`. Use the dedicated `registerWrapper` instead." );
```

Comment:
`not`??? what is this??? python? Haha.. I think we discussed "not" using the alternative tokens at some point. (pun intended). We can revisit this, but my preference would be to stick with one throughout the code....and that would be `! , && , ||`. I think that these are probably implemented as macro constants, which makes me like them less.

https://en.wikipedia.org/wiki/C_alternative_tokens

Reply:
Yes I will change this. I got this habit of using those english words instead.
Otherwise, is the patch OK?

Reply:
Fun fact, in the Wikipedia article it's said that "_The above-mentioned identifiers are [operator keywords](https://en.wikipedia.org/wiki/Operators_in_C_and_C%2B%2B) in the [ISO C++ programming language](https://en.wikipedia.org/wiki/ISO_C%2B%2B_programming_language) and do not require the inclusion of a header file._".

And code like `int not = 3;` won't compile. Which I never noticed.
```
error: expected unqualified-id before 'not' token
```